docs/demos/sequential_testing.ipynb

```diff
@@ -0,0 +1,3260 @@
+{
```

Comment:
<p>This output should be silenced, not sure why capture is not working here. This could be a fix:</p><p><a href="https://github.com/Lightning-AI/pytorch-lightning/discussions/6341" target="_blank">https://github.com/Lightning-AI/pytorch-lightning/discussions/6341</a></p><pre class="ql-syntax" spellcheck="false">import logging
logging.getLogger("pytorch_lightning").setLevel(logging.ERROR)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cnellington/Contextualized/pull/220/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>